api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/ManageLabelsForPersonGenerator.java

```diff
@@ -331,14 +334,14 @@ public int compare(HashMap<String, String> h1, HashMap<String, String> h2) {
 
 	private Object isEditable(VitroRequest vreq, EditConfigurationVTwo config) {
 		Individual individual = EditConfigurationUtils.getIndividual(vreq, config.getSubjectUri());
-		AddDataPropertyStatement adps = new AddDataPropertyStatement(
+		DataPropertyStatementAccessObject adps = new DataPropertyStatementAccessObject(
```

Comment:
I would change variable names to make sense with the new type (eg. `dpsAccessObject`).

Reply:
Done